Doc: Add a migration guide from Helm 2.1 to 3.0 (#2291)
grafana/mimir: docs/sources/operators-guide/deploying-grafana-mimir/upgrade-helm-chart-2.1-to-3.0/_index.md

---

> title: "TODO: Grafana Mimir Helm chart version 3.0 Upgrade Guide"

Reviewer: ```suggestion
title: "Uprade the Grafana Mimir Helm chart from version 2.1 to 3.0"
```

> menuTitle: "TODO: Helm chart v3.0"

Reviewer: ```suggestion
menuTitle: "Upgrade Helm chart 2.1 to 3.0"
```

> description: "Upgrade guide for the Grafana Mimir Helm chart version 3.0"

Reviewer: ```suggestion
description: "Uprade the Grafana Mimir Helm chart from version 2.1 to 3.0"
```

weight: 100
---

# TODO: Grafana Mimir Helm chart version 3.0 Upgrade Guide

> ## Breaking Changes

Reviewer: ```suggestion
## Breaking changes
```

> Several parameters available in version 2.1 of the mimir-distributed Helm chart have changed.

Reviewer: ```suggestion
Several parameters that were available in version 2.1 of the mimir-distributed Helm chart have changed.
```

> Follow these steps to migrate your Helm values file to 3.0.

Reviewer: ```suggestion
**To migrate your Helm values file to 3.0:**
```
